.github/CHANGELOG.md

```diff
@@ -522,6 +522,9 @@ Olivia Di Matteo, Ashish Panigrahi
 
 <h3>Improvements</h3>
 
+* Change the order of the covariance matrix and the vector of means internally
```

Comment:
Hi @l45k, this item now is listed for v0.16.0 of PennyLane, which we have released recently.

Therefore, two small things would be needed:
* Moving this entry up in the file and adding it to the `<h3>Improvements</h3>` section of the new, v0.17.0 release (~line 8),
* Adding yourself to the list of contributors, as that is a separate, new release with new contributors. You can do that on ~line 27.